sycl/plugins/unified_runtime/ur/ur.hpp

```diff
@@ -99,6 +101,100 @@ class SpinLock {
   std::atomic_flag MLock = ATOMIC_FLAG_INIT;
 };
 
+// The wrapper for immutable data.
+// The data is initialized only once at first access (via ->) with the
+// initialization function provided in Init. All subsequent access to
+// the data just returns the already stored data.
+//
+template <class T> struct ZeCache : private T {

```

Comment:
I may be wrong, but `Ze` seems like a leftover from L0-related naming. Should it be renamed to be more general?

Reply:
For sure, but renaming it here means re-naming uses in PI L0 plugin too, which might be big.
I'd prefer us to rename things that are completely moved to UR already.